recycling.md

```diff
@@ -0,0 +1,17 @@
+# Recycling
+
+[Terracycle](https://www.terracycle.com/) will recycle a number of materials not accepted by our municipal recycling system.
+
+The [Terracycle KW](https://www.facebook.com/groups/terracyclekw/) Facebook group has lots of useful information.
+
+REEP Green manages a [map](https://reepgreen.ca/exploring-the-circular-economy-in-waterloo-region/) of locations that collect various items.
+
+## Notes on recycling specific items in KW
+[Waste Whiz](https://www.regionofwaterloo.ca/en/living-here/the-waste-whiz.aspx) works fairly well.
+
+If you really can't figure something out, email [waste@regionofwaterloo.ca](mailto:waste@regionofwaterloo.ca).
+
+- Leave lids on plastic bottles with plastic caps.
+- Remove metal lids from glass bottles.
+- Tape must be removed from cardboard boxes.
+- Staples do not need to be removed from paper.
```

Comment:
Add date / dates for when this information was last confirmed.

Reply:
Done!